code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -36,77 +37,97 @@
 	work_damage_amount = 14
 	work_damage_type = BLACK_DAMAGE
 	/* Sounds */
-	projectilesound = 'sound/effects/attackblob.ogg'
-	attack_sound = 'sound/effects/attackblob.ogg'
-	deathsound = 'sound/effects/blobattack.ogg'
+	projectilesound = 'sound/abnormalities/meltinglove/ranged.ogg'
+	attack_sound = 'sound/abnormalities/meltinglove/attack.ogg'
+	deathsound = 'sound/abnormalities/meltinglove/death.ogg'
 	/*Vars and others */
 	loot = list(/obj/item/reagent_containers/glass/bucket/melting)
 	del_on_death = FALSE
-	var/mob/living/carbon/human/gifted_human = null
-	var/sanityheal_cooldown = 15 SECONDS
-	var/sanityheal_cooldown_base = 15 SECONDS
 	ego_list = list(/datum/ego_datum/weapon/adoration, /datum/ego_datum/armor/adoration)
 	gift_type =  /datum/ego_gifts/adoration
 	abnormality_origin = ABNORMALITY_ORIGIN_LOBOTOMY
 
+	var/mob/living/carbon/human/gifted_human = null
+	/// When FALSE - cannot attack or move
+	var/can_act = TRUE
+	/// Amount of BLACK damage done to all enemies around main target on melee attack. Also includes original target
+	var/radius_damage = 30
```

Comment:
90 Black is a lot of black, can you reduce this to 60-75?
High amounts of black damage is a lot worse than high amounts of red.

Reply:
90 damage is not a lot, so long as you have WAW+ armor(0.5).